### ISSUE TITLE
[Failed] [2.8.x] unresolved dependency: play sbt-plugin

### ISSUE THREAD

**Issue by ennru** [playframework/playframework#10244]
```
com.typesafe.play#sbt-plugin;2.8.1+65-a7339f98-akka-2.6.4+159-dc9f907c-akka-http-10.2.0-M1+7-e21c4199: not found
com.typesafe.play#sbt-scripted-tools;2.8.1+65-a7339f98-akka-2.6.4+159-dc9f907c-akka-http-10.2.0-M1+7-e21c4199: not found
```

https://travis-ci.com/github/playframework/playframework/jobs/324222531#L489

**Comment by raboof** [comment]
Looks like this is #10215

**Comment by mkurz** [comment]
Looks like fixed by #10263 or #10341. If not it will be fixed by #10855
